src/components/pegout/PegOutForm.vue

```diff
@@ -160,6 +164,97 @@ export default class PegOutForm extends Vue {
 
   isReadyToSign = false;
 
+  tourSteps = [
+    {
+      target: '#wallet-connection',
+      content: `Click to select the rsk wallet where your ${this.environmentContext.getRbtcTicker()} are stored`,
+      params: {
+        highlight: true,
+        isFirst: true,
+      },
+    },
+    {
+      target: '#amount-field',
+      content: `Input the amount you want to convert into ${this.environmentContext.getBtcTicker()}`,
+      params: {
+        highlight: true,
+        isLast: false,
+      },
+    },
+    {
+      target: '#max-btn',
+      content: 'If you want to send all your available balance in the wallet',
```

Comment:
I'll change it for "If you want to send all the available balance in your wallet." or even "Click here if you want to send all the available balance in your wallet."
(i.e., the order of _your_ and _the_)